cmd/kubeadm/app/util/crypto/crypto.go

```diff
@@ -0,0 +1,74 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package crypto
+
+import (
+	"github.com/pkg/errors"
+
+	"crypto/aes"
+	"crypto/cipher"
+	"crypto/rand"
+)
+
+// CreateRandBytes returns a slice with bytes
+func CreateRandBytes(size uint32) ([]byte, error) {
+	bytes := make([]byte, size)
+	if _, err := rand.Read(bytes); err != nil {
+		return nil, err
+	}
+	return bytes, nil
+}
+
+// EncryptBytes returns the data encrypted with the given key
+func EncryptBytes(data, key []byte) ([]byte, error) {
+	block, err := aes.NewCipher(key)
+	if err != nil {
+		return nil, err
+	}
+	gcm, err := cipher.NewGCM(block)
+	if err != nil {
+		return nil, err
+	}
+	nonce, err := CreateRandBytes(uint32(gcm.NonceSize()))
+	if err != nil {
+		return nil, err
+	}
+	return gcm.Seal(nonce, nonce, data, nil), nil
+}
+
+// DecryptBytes returns de data decrypted
+func DecryptBytes(data, key []byte) ([]byte, error) {
```

Comment:
I'm not sure if it's part of this pr scope, but I added it to use on tests.

Reply:
:+1: